samples/msal-node-samples/standalone-samples/README.md

```diff
@@ -1,12 +1,16 @@
 # MSAL Node Standalone Samples
 
-The sample applications contained in this directory are independent samples of MSAL Node usage, covering all of the authorization flows that MSAL Node currently supports. While all the samples include default configuration values that work out-of-the-box, their configuration is easily customizable to use your own Azure AD app registration.
+The sample applications contained in this directory are independent samples of MSAL Node usage, covering each of the authorization flows that MSAL Node currently supports. MSAL Node covers many situations, and most developers will only need one of these samples.
+
+## Should I use this sample:
```

Comment:
Maybe we should add a list of reasons of why people should use these samples instead of the one we use for automated tests. Otherwise we may want to remove this section?

Reply:
I'd prefer to defer this topic to later.  I think the section is fine for now.  I'm not fully sure if I can explain the automated testing sample in the way I was able to explain these.